packages/react-components/react-tree/stories/C_Layouts/LayoutsDefault.stories.tsx

```diff
@@ -86,20 +85,38 @@ export const Default = () => {
         >
           Content
         </TreeItemLayout>
+        <TreeItemActions>
+          <Button aria-label="Edit" appearance="subtle" icon={<FlagRegular />} />
+          <Menu>
+            <MenuTrigger disableButtonEnhancement>
+              <Button aria-label="More options" appearance="subtle" icon={<MoreHorizontal20Regular />} />
+            </MenuTrigger>
+
+            <MenuPopover>
+              <MenuList>
+                <MenuItem>New </MenuItem>
+                <MenuItem>New Window</MenuItem>
+                <MenuItem disabled>Open File</MenuItem>
+                <MenuItem>Open Folder</MenuItem>
+              </MenuList>
+            </MenuPopover>
+          </Menu>
+        </TreeItemActions>
         <Tree>
-          <TreeItem>
+          <TreeItem itemType="branch">
             <TreeItemLayout>level 2, item 1</TreeItemLayout>
             <Tree>
-              <TreeItem>
+              <TreeItem itemType="leaf">
                 <TreeItemLayout>level 3, item 1</TreeItemLayout>
               </TreeItem>
             </Tree>
           </TreeItem>
         </Tree>
       </TreeItem>
-      <TreeItem>
-        <TreeItemPersonaLayout
-          aside={
+      <TreeItem
+        aside={{
```

Comment:
do you still need aside as a slot now?, the `TableCellActions` component uses absolute positioning to make sure the actions are always rendered to the right most part of the cell.

If you did similar absolute positioning you could leave out aside completely and use JSX children

Reply:
Absolute positioning is not desirable for the Tree, we want both `actions` and `aside` content to be part of main content being rendered, to simulate something as an appended content and not as an over the top content.